workflow/artifacts/gcs/gcs.go

```diff
@@ -200,6 +200,11 @@ func listByPrefix(client *storage.Client, bucket, prefix, delim string) ([]strin
 	return results, nil
 }
 
+func (g *ArtifactDriver) OpenStream(a *wfv1.Artifact) (io.ReadCloser, error) {
+	// todo: this is a temporary implementation which loads file to disk first
+	return common.LoadToStream(a, g)
+}
+
```

Comment:
@juliev0 
Hi. I found a todo comment you left.
I'm wondering about your intention with the LoadToStream function.
I'd like to improve it. :)